users/views.py

```diff
@@ -69,7 +74,7 @@ def form_valid(self, form):
                 return super().form_invalid(form)
         else:
             form.save()
-        return redirect(reverse('home'))
+        return redirect('profile', pk=self.request.user.id)
```

Comment:
If we already has calling `super().form_invalid(form)` method i would prefer to use `super().form_valid(form)` too. The same as redirect method it will redirect us to `profile`
```suggestion
        return super().form_valid(form)
```